docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -105,16 +105,16 @@ You can also let Maven control the deployment of your application using the App
 
 First, add the plugin to your `pom.xml`:
 
-[source, xml]
+[source,xml,subs=attributes+]
```

Comment:
I don't think you want that? I see no attributes.

Reply:
Yes, not needed, removed.
I just copied it from another example while trying to understand why the `<n>` didn't works